docs/dabp/release-notes/index.md

```diff
@@ -4,16 +4,20 @@ This section contains release notes for Onegini DABP.
 
 In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [Onegini Support](https://support.onegini.com).
 
+## Release date: 2021-04-28
+* Removing a users last membership will now not remove users identity in CIM
+* Fixed failed login with external IdP (such as AzureAD)
+* Shows error message for phone-validation from CIM
+
 ## Release date: 2021-04-14
 * After removing a user from a group, if he is no longer of any group his account will be removed from CIM
 * Translation fixes and improvements
 * Minor UI improvements
-
-   - Color of the Profile icon is now configurable
```

Comment:
Please do not change this, readthedocs dont handle nested lists correcly when done this way.